Source/Core/Core/ConfigManager.cpp

```diff
@@ -301,6 +301,7 @@ void SConfig::SaveNetworkSettings(IniFile& ini)
   network->Set("SSLVerifyCertificates", m_SSLVerifyCert);
   network->Set("SSLDumpRootCA", m_SSLDumpRootCA);
   network->Set("SSLDumpPeerCert", m_SSLDumpPeerCert);
+  network->Set("NetworkTimeout", m_NetworkTimeout);
```

Comment:
Could you use the new config system for this please? Migration has been slow, and adding new settings to SConfig won't help...

(Add a ConfigInfo to Core/Config/MainSettings.h/cpp, then update Core/ConfigLoaders/IsSettingSaveable.cpp)